scripts/src/server.ts

```diff
@@ -0,0 +1,15 @@
+const PACKAGE_JSON = require('../../package.json');
+import {Server as KinServer} from "@kinecosystem/kin-sdk";
+
+export class Server extends KinServer {
```

Comment:
why we need this inheritance?
There's one place (`KinClient`) that instantiate `Server` where we need to add the headers and that's all.
it makes a confuse regarding the name (Server vs KinServer)